package.json

```diff
@@ -0,0 +1,3 @@
+{
+  "dependencies": {}
```

Comment:
This looks like a mistake? do we need this file?

Reply:
Probably accidently `yarn`ed in the main repo once and missed these added files, yeah